src/pages/_app.tsx

```diff
@@ -65,7 +69,7 @@ function MyApp({Component, pageProps}: AppProps) {
             const locale = appInfo.locale?.split('-')[0] ?? 'en';
 
             if (locale !== 'en' && localeRouteMap[locale]) {
-                router.push(`/${localeRouteMap[locale]}`);
```

Comment:
is this using next? we can use query parameter instead of appending the query parameters manually.